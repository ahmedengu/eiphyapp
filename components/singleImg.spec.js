import { mount } from 'enzyme';
import SingleImg from './singleImg';

function setup(props = {}) {
  return mount(<SingleImg img={{ img: { url: 'https://via.placeholder.com/150' } }} {...props} />);
}

describe('SingleImg Tests', () => {
  it('should work', () => {
    const actual = setup();
    expect(actual).toBeTruthy();
  });

  it('should match snapshot', () => {
    const actual = setup();
    expect(actual.html()).toMatchSnapshot();
  });
});
