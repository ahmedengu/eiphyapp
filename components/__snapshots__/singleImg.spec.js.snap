// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SingleImg Tests should match snapshot 1`] = `
<div class="img-viewer__img">
  <div>
    <img src="https://via.placeholder.com/150"
         alt
    >
  </div>
</div>
`;
